spec/filters/kv_spec.rb

```diff
@@ -1147,11 +1144,21 @@
       plugin.filter(event)
     end
     context 'when a custom tag is defined' do
-      let(:options) { super().merge("tag_on_failure" => "KV-ERROR")}
+      let(:options) { super().merge("tag_on_failure" => ["KV-ERROR"])}
```

Comment:
We want to keep this spec in place, to validate that old-style configs with a single string being given still work without being broken. If this spec fails, then our implementation breaks the user experience and we need to find an implementation that doesn't do so.

```suggestion
      let(:options) { super().merge("tag_on_failure" => "KV-ERROR")}
```

Reply:
Makes sense, thanks. Updated.